apps/test/unit/lib/levelbuilder/lesson-editor/LessonEditorTest.js

```diff
@@ -20,23 +36,36 @@ describe('LessonEditor', () => {
     };
   });
 
+  afterEach(() => {
+    restoreRedux();
+  });
+
+  const createWrapper = overrideProps => {
+    const combinedProps = {...defaultProps, ...overrideProps};
+    return mount(
+      <Provider store={store}>
+        <LessonEditor {...combinedProps} />
+      </Provider>
+    );
+  };
+
   it('renders default props', () => {
-    const wrapper = shallow(<LessonEditor {...defaultProps} />);
+    const wrapper = createWrapper({});
     expect(wrapper.contains('Lesson Name'), 'Lesson Name').to.be.true;
-    // expect(wrapper.contains('Lesson Overview'), 'Lesson Overview').to.be.true;
-    // expect(
-    //   wrapper.contains('Overview of the lesson for students'),
-    //   'student overview'
-    // ).to.be.true;
-    // expect(
-    //   wrapper.contains('The purpose of the lesson is for people to learn'),
-    //   'purpose'
-    // ).to.be.true;
+    expect(wrapper.contains('Lesson Overview'), 'Lesson Overview').to.be.true;
+    expect(
+      wrapper.contains('Overview of the lesson for students'),
+      'student overview'
+    ).to.be.true;
+    expect(
+      wrapper.contains('The purpose of the lesson is for people to learn'),
+      'purpose'
+    ).to.be.true;
```

Comment:
it looks like this is a repeat of content from https://github.com/code-dot-org/code-dot-org/pull/37282 but also isn't based directly off of that PR. I think the changes look good here but maybe just keep an eye out for merge conflicts in this file. since it is a test file you'll probably know if anything went wrong in the merge as long as entire test case doesn't go missing.

Reply:
Ya its the content from the other PR. I wanted to make sure the test still worked. Sorry I forgot to base off that PR